lib/matplotlib/backends/backend_agg.py

```diff
@@ -247,8 +247,7 @@ def get_text_width_height_descent(self, s, prop, ismath):
         d /= 64.0
         return w, h, d
 
-    @cbook._delete_parameter("3.2", "ismath")
-    def draw_tex(self, gc, x, y, s, prop, angle, ismath='TeX!', mtext=None):
+    def draw_tex(self, gc, x, y, s, prop, angle, mtext=None):
```

Comment:
after removing a `delete_parameter` the subsequent parameters also become keyword-only (because during the deprecation period the only way to pass them without triggering a warning would be to pass them as keyword, so that the deprecated parameter is not passed).